typescript/infra/config/environments/mainnet3/warp/configGetters/utils.ts

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert';
+
+import { ChainMap, ChainName, MovableTokenConfig } from '@hyperlane-xyz/sdk';
+import { arrayToObject, objMap } from '@hyperlane-xyz/utils';
+
+import { getRegistry } from '../../../../registry.js';
+import { WarpRouteIds } from '../warpIds.js';
+
+const REBALANCER = '0xa3948a15e1d0778a7d53268b651B2411AF198FE3';
+
+type RebalancingConfig = Required<
+  Pick<MovableTokenConfig, 'allowedRebalancingBridges' | 'allowedRebalancers'>
+>;
+
+export function getRebalancingBridgesConfigFor(
+  deploymentChains: readonly ChainName[],
+  chainsToExclude: readonly ChainName[] = [],
+): ChainMap<RebalancingConfig> {
+  const registry = getRegistry();
+  const mainnetCCTP = registry.getWarpRoute(WarpRouteIds.MainnetCCTP);
+
+  assert(mainnetCCTP, 'MainnetCCTP warp route not found');
+
+  const rebalanceableChains = deploymentChains.filter(
+    (chain) => !chainsToExclude.includes(chain),
+  );
+
+  const cctpBridgesByChain = Object.fromEntries(
+    mainnetCCTP.tokens.map(
+      ({ chainName, addressOrDenom }): [string, string] => {
+        assert(
+          addressOrDenom,
+          `Expected cctp bridge address to be defined on chain ${chainName}`,
+        );
+
+        return [chainName, addressOrDenom];
+      },
+    ),
+  );
+
+  return objMap(
+    arrayToObject(rebalanceableChains),
+    (currentChain): RebalancingConfig => {
+      const cctpBridge = cctpBridgesByChain[currentChain];
+      assert(cctpBridge, `No cctp bridge found for chain ${currentChain}`);
+
+      const allowedRebalancingBridges = Object.fromEntries(
+        deploymentChains
+          .filter((remoteChain) => remoteChain !== currentChain)
+          .map((remoteChain) => [remoteChain, [{ bridge: cctpBridge }]]),
+      );
```

Comment:
do we really want to add the bridge for destinations that are not "rebalanceable chains"?

Reply:
I have reasons to believe that chains that are not rebalanceable today might be in the future 👀 👀 
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/6664

Reply:
haha I think my other comment about deriving from the CCTP warp route is probably best for this